tools_docker/Debian_Uwsgi/Dockerfile

```diff
@@ -42,6 +42,7 @@ RUN \
     locales \
     mysql-client \
     poppler-utils \
+    qpdf \
```

Comment:
I suggest to put packages in alphabetical order, which will be easier to maintain.

Reply:
done.